lib/dao.js

```diff
@@ -934,6 +934,29 @@ DataAccessObject.all = function () {
   return DataAccessObject.find.apply(this, arguments);
 };
 
+/**
+ * Get settings via hiarchical determiniation
+ *
+ * @param {String} key The setting key
+ */
+DataAccessObject._getSetting = function(key) {
+  // Check for settings in model
+  var m = this.definition;
+  if(m && m.settings && m.settings[key]) {
+    return m.settings[key]
+  }
+
+  // Check for settings in connector
+  var ds = this.getDataSource();
+  if(ds && ds.settings && ds.settings[key]) {
+    return ds.settings[key];
+  }
+
+  // @TODO: Check for settings in app
```

Comment:
Just the TODO. It would be nice if you could send a patch to LoopBack which will apply app settings to newly created datasources, but that's not necessary to get this patch landed.